app/javascript/gobierto_data/webapp/components/sidebar/SidebarCategories.vue

```diff
@@ -79,22 +85,108 @@ export default {
       return ((this.filters[0] && (this.filters[0] || {}).count >= 1)
         || (this.filters[1] && (this.filters[1] || {}).count >= 1));
     },
-    filtersModify() {
-      return this.filters.length ? this.filters.map(d => ({ ...d, isToggle: true })) : []
+    isAllChecked() {
+      return this.routeItemsFrequency.length !== 0 || this.routeItemsCategory.length !== 0
```

Comment:
`length` is always numeric for objects, then you can just:

```suggestion
      return this.routeItemsFrequency.length || this.routeItemsCategory.length
```